authentication/subscription/service/oso_subscription_service_blackbox_test.go

```diff
@@ -273,3 +273,53 @@ func (s *osoSubscriptionServiceTestSuite) TestLoadOSOSubscriptionStatus() {
 	})
 
 }
+
+func (s *osoSubscriptionServiceTestSuite) TestDeactivateUser() {
+	// given
+	appRegURL := "https://some.osourl.io"
+	admin := "test-admin"
+	svcCtx := factory.NewServiceContext(s.Application, s.Application, nil, nil)
+	config := testsubscription.NewOSOSubscriptionServiceConfigurationMock(s.T())
+	config.GetOSORegistrationAppURLFunc = func() string {
+		return appRegURL
+	}
+	config.GetOSORegistrationAppAdminUsernameFunc = func() string {
+		return admin
+	}
+	config.GetOSORegistrationAppAdminTokenFunc = func() string {
+		return fmt.Sprintf("%s-token", admin)
+	}
+	svc := subscription.NewOSOSubscriptionService(svcCtx, config)
+
+	defer gock.OffAll()
+	gock.Observe(gock.DumpRequest)
+
+	s.Run("success", func() {
+		username := uuid.NewV4().String()
```

Comment:
Ideally, `username` will not be uuid but actual username like `myosio-preview`.  This test data is misleading.

Reply:
ok, I can change that to something like `user-<uuid>` in the test

Reply:
done in d770717